man/oomd.conf.xml

```diff
@@ -48,28 +48,30 @@
+        <term><varname>SwapUsedLimit=</varname></term>
+
+        <listitem><para>Sets the limit for swap usage on the system before <command>systemd-oomd</command>
+        will take action. If the fraction of swap used on the system is more than what is defined here,
+        <command>systemd-oomd</command> will act on eligible descendant cgroups, starting from the ones with
+        the highest swap usage to the lowest swap usage. Which cgroups are monitored and what action gets
+        taken depends on what the unit has configured for <varname>ManagedOOMSwap=</varname>.  Takes a value
+        specified in percent (when suffixed with "%"), permille ("‰") or permyriad ("‱"), between 0% and
+        100%, inclusive. Defaults to 90%.</para></listitem>
       </varlistentry>
 
       <varlistentry>
         <term><varname>DefaultMemoryPressureLimit=</varname></term>
 
-        <listitem><para>Sets the limit for memory pressure on the unit's cgroup before <command>systemd-oomd</command>
-        will take action. A unit can override this value with <varname>ManagedOOMMemoryPressureLimit=</varname>.
-        The memory pressure for this property represents the fraction of time in a 10 second window in which all tasks
-        in the cgroup were delayed. For each monitored cgroup, if the memory pressure on that cgroup exceeds the
-        limit set for longer than the duration set by <varname>DefaultMemoryPressureDurationSec=</varname>,
-        <command>systemd-oomd</command> will act on eligible descendant cgroups,
-        starting from the ones with the most reclaim activity to the least reclaim activity. Which cgroups are
-        monitored and what action gets taken depends on what the unit has configured for
-        <varname>ManagedOOMMemoryPressure=</varname>. Takes a percentage value between 0% and 100%, inclusive.
+        <listitem><para>Sets the limit for memory pressure on the unit's cgroup before
+        <command>systemd-oomd</command> will take action. A unit can override this value with
+        <varname>ManagedOOMMemoryPressureLimit=</varname>.  The memory pressure for this property represents
+        the fraction of time in a 10 second window in which all tasks in the cgroup were delayed. For each
+        monitored cgroup, if the memory pressure on that cgroup exceeds the limit set for longer than the
+        duration set by <varname>DefaultMemoryPressureDurationSec=</varname>, <command>systemd-oomd</command>
+        will act on eligible descendant cgroups, starting from the ones with the most reclaim activity to the
+        least reclaim activity. Which cgroups are monitored and what action gets taken depends on what the
+        unit has configured for <varname>ManagedOOMMemoryPressure=</varname>. Takes a value specified in
+        percent (when suffixed with "%"), permille ("‰") or permyriad ("‱"), between 0% and 100%, inclusive.
```

Comment:
Maybe
> "Takes a fraction specified in the same way as <varname>SwapUsedLimit=</varname> above. Defaults to 60%."

to avoid repeating the description?